packages/airnode-node/src/workers/index.ts

```diff
@@ -12,6 +12,13 @@ export function spawn(params: WorkerParameters): Promise<WorkerResponse> {
       return gcp.spawn(params);
 
     case 'local':
-      return localHandlers[params.functionName](params.payload);
+      switch (params.payload.functionName) {
```

Comment:
You needed to rewrite this because of TS complaints right?

Reply:
Yes.